Tests/StoreKitUnitTests/StoreKit2/StoreEnvironmentTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreEnvironmentTests.swift
+//
+//  Created by Mark Villacampa on 11/15/23.
+
+import Nimble
+@testable import RevenueCat
+import StoreKitTest
+import XCTest
+
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+@MainActor
+class StoreEnvironmentTests: StoreKitConfigTestCase {
```

Comment:
I don't think this needs to inherit `StoreKitConfigTestCase`, it can be a regular unit test (and moved to `UnitTests`), right?